deploy/tests/test_utils.py

```diff
@@ -1,37 +1,50 @@
 from pathlib import Path
 from unittest.mock import MagicMock, call
-from util.fsspec_utils import downloadURI
+from util.io import downloadURI
 from fsspec.implementations.local import LocalFileSystem
+from botocore.exceptions import ClientError
+import pytest
 
+pytest.mark.usefixtures("asw_test_credentials")
```

Comment:
```suggestion
pytest.mark.usefixtures("aws_test_credentials")
```

Reply:
I wonder how this didn't throw an exception....

Reply:
Yeah, me too.  Makes me think even more that we should just do the stuff in that fixture by default in conftest.py and swap the requirement to be so that people who want to really get out to AWS during pytest have to do work to undo the fake credentials.